index.html

```diff
@@ -55,7 +55,7 @@ <h1>報告書を、<br class="br-sp">もっと使いやすく。</h1>
                   <label><input type="checkbox" class="ministry" value="14">金融庁</label><br>
                 </div>
                 <input type="search" id="searchWord" name="" autocomplete="off" placeholder="検索">
-                <input type="submit" id="submit" name="">
+                <input type="submit" id="submit" name="" value="検索">
```

Comment:
ここは `input` タグよりも `button` タグのほうが意味的に良さそうです！

```suggestion
                <button id="submit" type="submit">検索</button>
```

Reply:
検索画面の検索バーはボタンの中身がなかったのでそちらも合わせて修正していただけるといいと思います！

[![Image from Gyazo](https://thumb.gyazo.com/thumb/200/eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpbWciOiJfMDYwOTE2ZjUxNTU0NGU3YmQzZmJiN2I2YzM3YWY5MTAifQ.P3jrrcWpc4pi8tym1DklZIiNG0ptHwd4Mqv1hh0VWPI-png.jpg)](https://gyazo.com/84d9ef07acfcc901cf3bb22429a4d8a8)

```
<input type="submit" id="submit2" name="">
```

Reply:
指摘ありがとう！今晩追加でプルリク出しやす！！